compiler/noirc_evaluator/src/ssa/ir/printer.rs

```diff
@@ -53,6 +53,9 @@ fn value(dfg: &DataFlowGraph, id: ValueId) -> String {
         Value::Intrinsic(intrinsic) => intrinsic.to_string(),
         Value::ForeignFunction(function) => function.clone(),
         Value::Param { .. } | Value::Instruction { .. } => id.to_string(),
+        Value::Global(_) => {
+            format!("@{id}")
+        }
```

Comment:
Can you change the format of globals to avoid the LLVM `@`? The rest of our IR is based on cranelift, and I think we can use our existing convention of changing the letter before the id instead. E.g. `g0`, `g1`, etc.

Reply:
I'll change to the letter. 

Just noting that in this case our globals are based more on LLVM. Global values in cranelift are _not_ known as compile time https://github.com/bytecodealliance/wasmtime/blob/main/cranelift/docs/ir.md#global-values. In LLVM global variables represent regions of memory allocated at compile time that must be initialized.

Reply:
Yep, I'm aware, just thought we should keep the same naming scheme

Reply:
I have updated the PR description with how the SSA looks now.